_docs/controller/configuration-options.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: Configuration
+permalink: /docs/controller/controller-configuration-options/
+category: Controller
+order: 3
+---
+
+The following instructions apply to both Docker-based and Kubernetes-based
+installations. The Amalgam8 Controller supports a number of configuration
+options, most of which are set through environment variables. The environment
+variables can be set via command line flags as well.
+
+The following environment variables are available. All of them are optional.
+
+| Environment Key | Flag Name                   | Description | Default Value |
+|:----------------|:----------------------------|:------------|:--------------|
+| A8_API_PORT | --api_port | API port | 8080 |
+| A8_CONTROL_TOKEN | --control_token | Controller API authentication token | ABCDEFGHIJKLMNOP |
+| A8_ENCRYPTION_KEY | --encryption_key | secret key | abcdefghijklmnop |
```

Comment:
Needs more description.. 
e.g., secret key used to encrypt data stored in the database.